solution/ui/regulations/eregs-vite/src/main.js

```diff
@@ -1,47 +1,39 @@
-import Vue from "vue";
-import VueRouter from "vue-router";
-
-const { isNavigationFailure, NavigationFailureType } = VueRouter;
-
+import { createApp } from "vue";
+import { isNavigationFailure, NavigationFailureType } from "vue-router";
+import Clickaway from "directives/clickaway";
 import vuetify from "./plugins/vuetify";
+
 import App from "./App.vue";
 import vueRouter from "./router";
 
-import Clickaway from "./directives/clickaway";
-
 const mountEl = document.querySelector("#vite-app");
-Vue.config.devtools = true;
 const { customUrl, host } = mountEl.dataset;
 
 let { isAuthenticated } = mountEl.dataset;
 isAuthenticated = isAuthenticated === "True";
 
-Vue.directive("clickaway", Clickaway);
+const app = createApp(App, { ...mountEl.dataset });
+app.use(vuetify);
+
+app.directive("clickaway", Clickaway);
 
 const router = vueRouter({ customUrl, host });
 
-router.beforeEach((to, _from, next) => {
+router.beforeEach((to) => {
     const pageTitle = "Find by Subject | Medicaid & CHIP eRegulations";
 
     if (to.name === "subjects") {
-        if ( window.event?.type === "popstate" ) {
+        if (!to.query?.subject) {
             document.title = pageTitle;
-        } else if (_from.name) {
-            if (to.params?.subjectName) {
-                // set document title here with available information
-                document.title = `${to.params.subjectName} | ${pageTitle}`;
-            } else {
-                document.title = pageTitle;
-            }
```

Comment:
passing both `query` and `params` into a route is no longer supported in Vue Router, so this logic has been moved into the `Subjects.vue` component